src/Helper/MyParcelCollection.php

```diff
@@ -426,9 +451,13 @@ public function setLinkOfLabels($positions = self::DEFAULT_A4_POSITION)
                     implode(';', $conceptIds) . '/' . $this->getRequestBody(),
                     MyParcelRequest::REQUEST_HEADER_RETRIEVE_LABEL_LINK
                 )
-                ->sendRequest('GET', MyParcelRequest::REQUEST_TYPE_RETRIEVE_LABEL);
+                ->sendRequest('GET', $requestType);
+
+            if ($this->use_label_prepare) {
```

Comment:
Dan kan je beter hier ook $this->useLabelPrepare() doen. Dan kan use_label_prepare weg als field. Dat scheelt weer complexiteit.